packages/common/test/directives/ng_optimized_image_spec.ts

```diff
@@ -1371,11 +1390,21 @@ describe('Image directive', () => {
         expect(consoleWarnSpy.calls.argsFor(0)[0]).toMatch(
           new RegExp(`NG0${RuntimeErrorCode.PLACEHOLDER_DIMENSION_LIMIT_EXCEEDED}:`),
         );
+
+        globalThis['ngServerMode'] = undefined;
       });
     }
   });
 
   describe('preconnect detector', () => {
+    beforeAll(() => {
+      globalThis['ngServerMode'] = true;
+    });
+
+    afterAll(() => {
```

Comment:
I think it'd be great to do `beforeEach` and `afterEach` here instead, so we minimize a chance of leaking the `ngServerMode` global state:
```suggestion
    beforeEach(() => {
      globalThis['ngServerMode'] = true;
    });

    afterEach(() => {
```